src/main/kotlin/com/linchpino/core/service/EmailService.kt

```diff
@@ -0,0 +1,82 @@
+package com.linchpino.core.service
+
+import com.linchpino.core.entity.Interview
+import com.linchpino.core.enums.AccountStatusEnum
+import jakarta.mail.internet.InternetAddress
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.core.env.Environment
+import org.springframework.mail.javamail.JavaMailSender
+import org.springframework.mail.javamail.MimeMessageHelper
+import org.springframework.stereotype.Service
+import org.thymeleaf.context.Context
+import org.thymeleaf.spring6.SpringTemplateEngine
+
+@Service
+class EmailService(
+    private val emailSender: JavaMailSender,
+    private val templateEngine: SpringTemplateEngine,
+    private val environment: Environment
+) {
+    @Value("\${application.url}")
+    var applicationUrl: String? = null
+
+    fun sendingInterviewInvitationEmailToJobSeeker(interview: Interview) {
+        val fullName = "${interview.jobSeekerAccount!!.firstName} ${interview.jobSeekerAccount!!.lastName}"
```

Comment:
Use !! in production code with more caution. These objects are all nullable and may throw null pointer exception. Unless you are sure that these objects never take null value.